client/status/v2.go

```diff
@@ -114,19 +115,18 @@ func FromStatusV2(st *status.Status) any {
 	return decoder
 }
 
-// ToStatusV2 converts instance to status.Status message structure. Inverse to [FromStatusV2] operation.
+// ToStatusV2 converts error to status.Status message structure. Inverse to [FromStatusV2] operation.
 //
 // If argument is the [StatusV2] instance, it is converted directly.
 // Otherwise, successes are converted with [status.OK] code w/o details and message,
 // failures - with [status.Internal] and error text message w/o details.
-func ToStatusV2(st any) *status.Status {
-	if v, ok := st.(StatusV2); ok {
-		return v.ToStatusV2()
+func ToStatusV2(err error) *status.Status {
+	if err == nil {
+		return newStatusV2WithLocalCode(status.OK, status.GlobalizeSuccess)
 	}
 
-	err, isErrorExists := st.(error)
-	if !isErrorExists {
-		return newStatusV2WithLocalCode(status.OK, status.GlobalizeSuccess)
+	if v, ok := err.(StatusV2); ok {
```

Comment:
`errors.As(err, StatusV2Instance)`, the error can be wrapped (and it should be tested).

Reply:
done